include/audio_modules/lc3_decoder.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright(c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#ifndef _LC3_DECODER_H_
+#define _LC3_DECODER_H_
+
+#include "audio_defines.h"
+#include "audio_module/audio_module.h"
+
+#define LC3_DECODER_MAX_FRAME_SIZE_MS 10
+
+#define LC3_DECODER_PCM_NUM_BYTES_MONO                                                             \
+	((CONFIG_LC3_DECODER_SAMPLE_RATE_HZ * CONFIG_LC3_DECODER_BIT_DEPTH_OCTETS *                \
+	  LC3_DECODER_MAX_FRAME_SIZE_MS) /                                                         \
+	 1000)
+#define LC3_DEC_TIME_US 1500
+
+/**
+ * @brief Private pointer to the module's parameters.
+ */
+extern struct audio_module_description *lc3_decoder_description;
```

Comment:
I might just be missing it in PR view, but I can't see this being used - is it supposed to carry information about the current decoder to the next module?

Reply:
I is passed to the audio_open_open() function and supplies all the information needed to execute the modules core function.